backend/program/scrapers/__init__.py

```diff
@@ -19,32 +21,52 @@ def __init__(self, hash_cache):
         self.settings = settings_manager.settings.scraping
         self.hash_cache = hash_cache
         self.services = {
-            Annatar: Annatar(self.hash_cache),
-            Torrentio: Torrentio(self.hash_cache),
-            Knightcrawler: Knightcrawler(self.hash_cache),
-            Orionoid: Orionoid(self.hash_cache),
-            Jackett: Jackett(self.hash_cache),
-            TorBoxScraper: TorBoxScraper(self.hash_cache),
-            Mediafusion: Mediafusion(self.hash_cache)
+            "Annatar": Annatar(self.hash_cache),

```

Comment:
use the class as the key

Reply:
But then how can i log the name of it? Is it something like `service.__name__`?

Reply:
its either `service.__name__` or `service.__class__.__name__ `

I cant remember